src/core_modules/capture-core/metaDataStoreLoaders/context/context.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import { StorageController } from 'capture-core-utils/storage';
+import type { QueryApiFn } from '../loader.types';
+
+let context;
+
+type StoreNames = {
```

Comment:
Could we also move those types on a separate file? I think its a good think to be doing and we end up having more clear files in the end. 

What do you think on that?

Reply:
Sure. What I've been doing is extracting them to a new file when the types are required in multiple files. But I agree it's cleaner if we always keep them in a separate file.

Reply:
That makes sense. Lets then leave them here since its not being reused